extensions/machine-learning/src/views/models/manageModels/modelImportLocationPage.ts

```diff
@@ -99,6 +99,13 @@ export class ModelImportLocationPage extends ModelViewBase implements IPageView,
 		}
 
 		if (this.importTable && this._labelComponent) {
+
+			// Add table name to the models imported.
+			// Since Table name is picked last as per new flow this hasn't been set yet.
+			if (this.modelsViewData && this.modelsViewData.length > 0) {
+				this.modelsViewData?.map(x => x.targetImportTable = this.importTable);
```

Comment:
This is already in an if statement that checks that modelsViewData exists so is unnecessary.

You can just shorten this whole thing to 
`this.modelsViewData?.forEach(x => x.targetImportTable = this.importTable);`

(using map here isn't really necessary - map is for if you're converting items into another form)

Reply:
you should use foreach here instead of map